src/oclHashcat.c

```diff
@@ -797,7 +797,7 @@ void status_display_automat ()
     hc_clGetEventProfilingInfo (data.ocl, device_param->event, CL_PROFILING_COMMAND_START, sizeof (time_start), &time_start, NULL);
     hc_clGetEventProfilingInfo (data.ocl, device_param->event, CL_PROFILING_COMMAND_END,   sizeof (time_end),   &time_end,   NULL);
 
-    const double total_time = (time_end - time_start) / 1000000.0;
+    const double total_time = ((double)(time_end - time_start) * (double)(device_param->device_timer_resolution)) / 1000000.0;
 
     fprintf (out, "%f\t", total_time);
```

Comment:
Sorry if I'm totally confused here, but is this really correct? I think it isn't.

Before this patch:
- If a device with 1 ns resolution gave a `time_end - time_start` resulting in 8000, we know it took 8 microseconds +/- one nanosecond.

- If a device with 80 ns resolution gave `time_end - time_start` resulting in 8000, we know it took 8 microseconds +/- 80 nanoseconds.

But after this patch, the latter will show up like... 8000*80 == 640000 meaning 640 microseconds? That's just not right.